xbmc/interfaces/generic/ScriptRunner.cpp

```diff
@@ -33,39 +33,29 @@ ADDON::AddonPtr CScriptRunner::GetAddon() const
 CScriptRunner::CScriptRunner() : m_scriptDone(true)
 { }
 
-bool CScriptRunner::StartScript(const ADDON::AddonPtr& addon, const std::string& path)
-{
-  return RunScriptInternal(addon, path, 0, false);
-}
-
 bool CScriptRunner::RunScript(const ADDON::AddonPtr& addon,
-                              const std::string& path,
+                              const CURL& url,
                               int handle,
                               bool resume)
 {
-  return RunScriptInternal(addon, path, handle, resume, true);
+  return RunScriptInternal(addon, url, handle, resume, true);
 }
 
 void CScriptRunner::SetDone()
 {
   m_scriptDone.Set();
 }
 
-int CScriptRunner::ExecuteScript(const ADDON::AddonPtr& addon, const std::string& path, bool resume)
+int CScriptRunner::ExecuteScript(const ADDON::AddonPtr& addon, const CURL& path, bool resume)
```

Comment:
Should be `const CURL& url` for consistency?

Reply:
If the answer is no, we'll merge now. If yes, go ahead and push an update then we'll merge.